src/scripts/main.js

```diff
@@ -1,3 +1,57 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const field = document.querySelector('.field').children[0];
+const rows = field.children;
+
+document.addEventListener('click', (e) => {
+  if (e.target === appendRow) {
```

Comment:
Switch case will be more appropriate in this case. Or you can create separate event listeners for 4 buttons. It's up to you